neuro-cli/src/neuro_cli/formatters/jobs.py

```diff
@@ -269,6 +276,21 @@ def __call__(self, job_status: JobDescription) -> RenderableType:
         return table
 
 
+class LifeSpanUpdateFormatter:
+    def __init__(self, datetime_formatter: DatetimeFormatter) -> None:
+        self._datetime_formatter = datetime_formatter
+
+    def __call__(self, job_status: JobDescription) -> RenderableType:
+        res = (
+            f"Updated job {job_status.id} life span to "
+            f"be {format_life_span(job_status.life_span)}"
+        )
+        life_span_ends = get_lifespan_ends(job_status)
+        if life_span_ends is not None:
+            res += f"\nLife ends (approx): {self._datetime_formatter(life_span_ends)}"
```

Comment:
```suggestion
            res += f"\nLife span ends (approx): {self._datetime_formatter(life_span_ends)}"
```
To be in sync with the text above